ledger/ledger_test.go

```diff
@@ -1579,6 +1581,12 @@ func TestListAssetsAndApplications(t *testing.T) {
 
 func TestLedgerMemoryLeak(t *testing.T) {
 	t.Skip() // for manual runs only
+	savedBaseAlgo := baseAlgoPerAccount
+	baseAlgoPerAccount = 10000000000
```

Comment:
this might cause data races if we run two tests that access this variable concurrently.

Reply:
passing this parameter in is the "safe" approach.

Reply:
this test is for manual running only (see `t.Skip()`) so I think it is ok to use a global var for it. Passing as a param is safer, agree, but requires 22 modifications that are not really necessary in all these cases.

Reply:
done, moved to args.